roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -118,7 +118,7 @@
   when:
   - openshift_logging_facts.elasticsearch.deploymentconfigs.keys() | count > 0
 
-# Create any new DC that may be required
+Create any new DC that may be required
```

Comment:
Does this work?  it looks like it should begin with either a '-' or '#'

Reply:
We need to add that comment back in. I think this would give a parsing error

Reply:
Typo, sorry  for that